core/src/main/java/org/testcontainers/containers/image/ImageData.java

```diff
@@ -0,0 +1,34 @@
+package org.testcontainers.containers.image;
+
+import com.github.dockerjava.api.command.InspectImageResponse;
+import com.github.dockerjava.api.model.Image;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor(access = AccessLevel.MODULE)
+public class ImageData {
+
+    public static ImageData from(Image image) {
+        return new ImageData(image.getCreated(), image.getId(), image.getParentId(), Arrays.asList(image.getRepoTags()), new ArrayList<>(), image.getSize(), image.getVirtualSize());
```

Comment:
please split arguments line-by-line after you move it to `DockerJavaImageData`'s constructor